src/components/patterns/link.tsx

```diff
@@ -44,10 +44,14 @@ export const DrupalActionLink = ({href, children, ...props}: PropsWithChildren<{
     <Link
       href={href}
       {...props}
-      className={twMerge("relative pr-30 hocus:text-brick no-underline rs-mt-neg1 pt-10", props.className)}
+      className="pt-10"
     >
-      {children}
-      <ChevronRightIcon className="inline absolute top-0 right-0 h-full pt-10"/>
+      <span className={twMerge("relative pr-30 no-underline rs-mt-neg1 hover:bg-black-10 hover:text-brick hocus:underline focus:text-brick active:text-brick", props.className)}
```

Comment:
we should actually put these classes on the link itself because when focusing, the focus happens on the link, not the span. 
Also, I believe you can combine the `hover:text-brick focus:text-brick`, into one `hocus`. but I recall the `twMerge` didn't work as well with that.

Reply:
The problem with putting the classes on the link is... well. Here's how it looks. 

![Screenshot 2024-02-14 at 6 09 13 PM](https://github.com/SU-SWS/sulgryphon-nextjs/assets/8387457/c6740458-e527-4e18-b5c0-582b94d822ad)

Reply:
You can definitely split up the classes. Any hover or focus classes must be on the link. So really the colors and the underline need to move. The others can stay here. But the `twMerge` should happen at the outer most element with the `props.className`. That allows passing a className prop down from a parent to make any desired adjustments.

Reply:
I said that and then got the classes to work by removing `pt-10` from the chevron icon. 

![Screenshot 2024-02-14 at 6 14 45 PM](https://github.com/SU-SWS/sulgryphon-nextjs/assets/8387457/e6f737f6-63c6-494c-89e0-ab67538f3f9c)

But it worries me to remove the padding-top because I wonder why it was placed there to begin with. I've play around with it and it seems fine.

Reply:
I settled on removing the padding top, unless you can see a reason to keep it. 

The `hocus` seems to work. 

I aligned its states with inline links. Seems to be right.

Reply:
Looks good. But you committed the console log again

Reply:
Argh. Too many control + z